src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -51,9 +51,7 @@ public static void loadModElements() {
 				new ModElementType<>("enchantment", 'm', BaseType.OTHER, RecipeType.NONE, EnchantmentGUI::new,
 						Enchantment.class));
 		ModElementType.FLUID = register(
-				new ModElementType<>("fluid", 'f', BaseType.BLOCK, RecipeType.BUCKET, FluidGUI::new, Fluid.class));
-		ModElementType.FUEL = register(
-				new ModElementType<>("fuel", null, BaseType.OTHER, RecipeType.NONE, FuelGUI::new, Fuel.class));
+				new ModElementType<>("fluid", 'u', BaseType.BLOCK, RecipeType.BUCKET, FluidGUI::new, Fluid.class));
```

Comment:
`u` is used by function MET right below this line, also this is not the PR that adds feature MET. Change this key back.